### DISCUSSION TITLE
Twitter view(s)

### DISCUSSION THREAD

**Discussion by mtfoley** [open-sauced/open-sauced#1094] [Ideas]
Feels like it might be good in open-sauced dashboard to have another source of recommendations in the Twitter handle @goodfirstissues. On  another note, we could additionally do something similar and show a Twitter feed in a goal view if it's found in the repo's description

**Comment by 0-vortex** [comment]
I like this - could use some onegraph stuff :D

**Reply by mtfoley** [reply]
I had thought the same. My question would be whether it could be done with dynamic queries (don't see why not, since I'm assuming  the Twitter data is public)

**Reply by mtfoley** [reply]
Just because that'd make it easier to contribute too.

**Reply by 0-vortex** [reply]
let me push some questionable implementation for #950 - maybe that helps seeing what queries we can use

**Reply by mtfoley** [reply]
lol if you do, I hope the gif involves that song "dirty deeds done dirt cheap"
